src/course-home/data/__snapshots__/redux.test.js.snap

```diff
@@ -528,6 +528,7 @@ exports[`Data layer integration tests Test fetchOutlineTab Should fetch, normali
               "hideFromTOC": undefined,
               "icon": null,
               "id": "block-v1:edX+DemoX+Demo_Course+type@sequential+block@bcdabcdabcdabcdabcdabcdabcdabcd1",
+              "isPreview": undefined,
```

Comment:
Technically, this is inaccurate. The backend will always send True / False. Should probably update, if just for clarity.